presto-docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -90,6 +90,25 @@ following aggregate functions:
 * :func:`var_pop`
 * :func:`var_samp`
 
+Table property ``data_compression``
+-----------------------------------
+
+With the ``data_compression`` table property you can specify the data compression
```

Comment:
Turn the sentence around

You can specify the data compression policy for SQL Server tables with the ...